pliers/converters/misc.py

```diff
@@ -13,6 +13,8 @@ class ExtractorResultToSeriesConverter(Converter):
 
     def _convert(self, result):
         df = result.to_df(timing=False, metadata=False, object_id=False)
```

Comment:
I still don't get why `object_id=False` would not guarantee that `object_id` is not in the columns. Is it because `to_df` could be a custom function for the Extractor that does not respect that argument?

Reply:
It's because if a custom `_to_df` function is defined for an Extractor, `to_df` calls that first. If the custom function also adds an `object_id` column, there is no place in the code where that column is dropped in the `to_df` call. 
The object_id argument only makes it so that an object_id column is _added_ if not present.

Alternative solutions to fixing the problem this way could be to actually edit the to_df function so to drop `object_id` column if present when `object_id=False`.

Another option is changing the custom `_to_df` function for `BertLMExtractor`. 
At the moment, it inherits from `BertExtractor`, where adding a custom `object_id` is necessary. But  I guess we could define a `_to_df` for` BertLMExtractor` where `object_id` is not added.

Open to implement any of these if more elegant.

Reply:
I think the second option is a good idea but it's out of the scope of this PR so let's merge but and deal with it later.